backend/app/Chat/User/UserLock.php

```
@@ -0,0 +1,201 @@
+            $stmt->execute([':uuid' => $userUuid]);
+            $user = $stmt->fetch(\PDO::FETCH_ASSOC);
+
+            if (!$user) {
+                $pdo->rollBack();
+
+                return false;
+            }
+
+            // Check if user is locked and if lock has expired
+            if ($user['locked'] === 'true') {
+                $lastSeen = new \DateTime($user['last_seen']);
```

PHPMD (2.15.0)

Missing class import via use statement (line '52', column '33'). (Clean Code Rules)

(MissingImport)